### ISSUE TITLE
highlighting when selecting should be optional

### ISSUE THREAD

**Issue by Playzinho** [facebookincubator/RapiD#593]
### Description

[![Captura-de-tela-de-2022-09-25-17-19-30.png](https://i.postimg.cc/KzjLcYgG/Captura-de-tela-de-2022-09-25-17-19-30.png)](https://postimg.cc/R6rqX4Ry)

I understand that it may be necessary for new users but I hope it is optional 

### Version

2.0-alpha

### What browser are you seeing the problem on? What version are you running?

Chrome v105.0

### The browser URL at the time you encountered the bug

https://mapwith.ai/rapid-v2-alpha#background=Maxar-Premium&datasets=fbRoads,msBuildings&disable_features=boundaries&id=w531126040&map=20.02/-4.83598/-40.32848

**Comment by StephanGeorg** [comment]
I agree. It's often necessary to rotate the footprint. With the highlighting in v2.0 it's hard to see the underlying basemap.

**Comment by bhousel** [comment]
Yes, the current hover and highlight styles are not at all finalized - we'll be improving this in upcoming releases.
